syntax = "proto3";

package logging;

service LoggingService {
    rpc LogMessage(LogRequest) returns (LogResponse);
    rpc GetLogs(Empty) returns (LogsResponse);
}

message LogRequest {
    string id = 1;
    string msg = 2;
}

message LogResponse {
    string detail = 1;
}

message Empty {}

message LogsResponse {
    repeated string logs = 1;
}
